packages/detector.go

```diff
@@ -11,67 +11,143 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+// Package packages implements package manager detection and package
+// manager specific output formatting.
```

Comment:
This comment reads a little funny to me. #youGetAPackageAndYouGetAPackage :)

Reply:
Removed. Getting too OCD with linting warnings :-D